pom.xml

```diff
@@ -552,7 +564,6 @@ Bundle-DocURL: https://github.com/Netcentric/accesscontroltool
                     <plugin>
                         <groupId>org.owasp</groupId>
                         <artifactId>dependency-check-maven</artifactId>
-                        <version>6.2.0</version>
```

Comment:
Is this intentional? Repeatable builds need fixed plugin versions.

Reply:
It is fixed in https://github.com/Netcentric/accesscontroltool/pull/597/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R398.